### DISCUSSION TITLE
[Feature] Normalize Duplicate Detection Factor: 0-100 instead of 0.001 to 0.1

### DISCUSSION THREAD

**Discussion by radh21301** [immich-app/immich#11837] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I think it would be more intuitive to have a score ranging from 0-100.  According to the help info: It says 0.001 gives only very close ones, and 0.1  might give false positives.

So the numbering would be something like:
0: corresponding to old convention 0.1
100: corresponds to old convention 0.001

This normalizes the range from 0 to 100, which I think is more intuitive. But maybe in the future, immich will use another model, and the user needs to give a number from, say, 0.7 to 1.8 -> normalizing would still work. The end user does not need to know the specifics. All they need to know: Close to 100 means very similar. Close to 0 means you will get a lot of false positives.

Digikam also does it like that - though I am unsure what algorithm they use. 
<img width="295" alt="image" src="https://github.com/user-attachments/assets/b4c4d842-6ab9-43c8-99d0-097b80e68e4e">

*It does not necessarily need to be 0-100. Something like 0-10 also works. As long as the scale is normalized*

Edit: On second thought, I think using 0-10 is better than 0-100. Because when I see the number 100, the first thing that comes to my mind is that the images are the exact same - meaning ditto copies. But since we are using CLIP, saying that two things are ditto copies might be false?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by mertalev** [comment]
My first thought is that it’s misleading to present distance as a percentage since they’re fundamentally different things. 1-10 is also too coarse: there’s a huge difference between 1 and 2. 1-1000 could maybe work. But again, I should emphasize that this is about the extent of similarity, not a % chance that they’re identical.

**Reply by radh21301** [reply]
I completely agree. I first thought 0-100, but then realized that is not how the duplicate feature using CLIP works. But some form of normalization would be easy for the end user I think. 0-1000 is also works.
